File: Back/.Net/BetFriends.Domain/Events/BetAnswered.cs
using BetFriends.Domain.Bets;
using BetFriends.Domain.Members;
using BetFriends.Shared.Domain;

namespace BetFriends.Domain.Events;

public record BetAnswered(BetId BetId, bool Answer, MemberId MemberId) : IDomainEvent;

File: Back/NodeJS/modules/bets/src/domain/answerBets/events/BetAnswered.ts
import { BetId } from "../../bets/BetId";
import { IDomainEvent } from "../../IDomainEvent";
import { MemberId } from "../../members/MemberId";

export class BetAnswered implements IDomainEvent {
    constructor(public readonly BetId: BetId,
                public readonly GamblerId: MemberId,
                public readonly Answer: boolean
    ) {}
    Type: string = BetAnswered.name;
}